visualization/app/codeCharta/ui/dialog/dialog.globalSettings.component.spec.ts

```diff
@@ -0,0 +1,124 @@
+import "./dialog.module"
+import "../codeMap/codeMap.module"
+import { DialogGlobalSettingsController } from "./dialog.globalSettings.component"
+import { SettingsService } from "../../state/settingsService/settings.service"
+import { Settings } from "../../codeCharta.model"
+import { instantiateModule, getService } from "../../../../mocks/ng.mockhelper"
+import { DEFAULT_SETTINGS } from "../../util/dataMocks"
+import { IRootScopeService } from "angular"
+import _ from "lodash"
+
+describe("DialogGlobalSettingsController", () => {
+	let dialogGlobalSettingsController: DialogGlobalSettingsController
+	let $mdDialog
+	let settingsService: SettingsService
+	let $rootScope: IRootScopeService
+	let settings: Settings
+
+	beforeEach(() => {
+		restartSystem()
+		withMockedSettingsService()
+		rebuildController()
+	})
+
+	function restartSystem() {
+		instantiateModule("app.codeCharta.ui.dialog")
+
+		$mdDialog = getService("$mdDialog")
+		settingsService = getService<SettingsService>("settingsService")
+		$rootScope = getService<IRootScopeService>("$rootScope")
+
+		settings = _.cloneDeep(DEFAULT_SETTINGS)
+	}
+
+	function rebuildController() {
+		dialogGlobalSettingsController = new DialogGlobalSettingsController($mdDialog, $rootScope, settingsService)
+	}
+
+	function withMockedSettingsService(mockSettings: Settings = settings) {
+		settingsService = jest.fn<SettingsService>(() => {
+			return {
+				getSettings: jest.fn().mockReturnValue(mockSettings)
+			}
+		})()
+	}
+
+	function setEmptyViewModel() {
```

Comment:
We don't need to set an empty view model because it's empty on construction anyway.

Remove this function and it's calls.

Reply:
True now we don't need it anymore.